rel-eng/atomic-enterprise-master.service

```diff
@@ -3,14 +3,12 @@ Description=Atomic Enterprise Master
 Documentation=https://github.com/openshift/origin
 After=network.target
 After=etcd.service
-Before=atomic-enterpries-node.service
-Before=atomic-enterprise-sdn-node.service
-Before=atomic-enterprise-sdn-master.service
+Before=atomic-enterprise-node.service
 Requires=network.target
 
 [Service]
 Type=notify
-EnvironmentFile=/etc/sysconfig/atomic-enterprise-master
+EnvironmentFile=/etc/sysconfig/atomic-openshift-master
 Environment=GOTRACEBACK=crash
 ExecStart=/usr/bin/atomic-enterprise start master --config=${CONFIG_FILE} $OPTIONS
```

Comment:
The binary is showing up in /bin/. Either this should be updated for /bin/ or the spec should link/lay down to /usr/bin/.